README.md

```diff
@@ -129,6 +133,62 @@ pipelines: # a list of pipelines to run
         value: 6
 ```
 
+The configuration can be dynamically overridden by providing a dictionary of overrides to the runner. All overrides must adhere to configurations schema, with pipeline.extras section available for custom schema.
+Here are few examples of overrides:
+
+#### Simple override of a single value
+Specify the path to the value in the configuration file as a dot-separated string
+
+```python
+Runner(
+        spark,
+        config_path="tests/overrider.yaml",
+        run_date="2023-03-31",
+        overrides={"runner.watch_period_value": 4},
+    )
+```
+
+#### Override list element
+You can refer to list elements by their index (starting with 0)
+```python
+overrides={"runner.mail.to[1]": "a@b.c"}
```

Comment:
maybe confusing to say start at 0 but use 1 as the example :P also do you REALLY need to state starting index?